dashboards-observability/public/components/explorer/explorer.tsx

```diff
@@ -1110,28 +1102,21 @@ export const Explorer = ({
     }
   };
 
+  const stopLive = () => {
+    setLiveTailName('Live');
+    setIsLiveTailOn(false);
+    setLiveHits(0);
+    setIsLiveTailPopoverOpen(false);
+    if (isLiveTailOnRef.current) setToast('Live tail Off', 'danger');
```

Comment:
Question: does this needs to be a red colored toast. Would this give an erroneous impression to users?

Reply:
this was a change proposed by David to change it to red so that it would be different from the On toast message